### PR TITLE
Rework of HPE Storage Plugin GitHub Page

### PR DESCRIPTION
**Pull request by budhac**
playbook for uninstalling the plugin and cleanup of files

### CODE REVIEW

File: docs/system-reqs.md [hunk 1]
```diff
@@ -3,24 +3,24 @@
 The v1.0 or later version of 3PAR Volume plugin of Docker Engine Managed plugin system is supported and available now on Docker store (https://store.docker.com/plugins/hpe-3par-docker-volume-plugin/).
 Versions starting at v1.0 have been tested and are supported on the following Linux distributions:
 
-- Ubuntu 16.04
```
Comment (prablr79) [inline docs/system-reqs.md]: I would suggest not to keep later word until unless we qualify and certify the OS

File: docs/usage.md [hunk 2]
```diff
@@ -1,65 +1,109 @@
-## Usage
```
Comment (prablr79) [inline docs/usage.md]: This is live help document which will be part of the code path , we need to keep as is

**Comment by budhac** [comment]
All customer related documents have been moved under docs and not under the root directory in order to keep the main landing clean and improve navigation of the site. The root docs folders holds the basic customer install documents, any advanced topics/documents were moved under the /docs/advanced.

File: Dockerfile [hunk 3]
```diff
@@ -1,84 +0,0 @@
-FROM alpine:3.4
```
[the remark this answers is not in the document]
Comment (wdurairaj) [reply Dockerfile]: @budhac

Dockerfile is part of the source which builds the container. I don't approve this file being touched for a doc change.

[the remark this answers is not in the document]
Comment (budhac) [reply Dockerfile]: Then it needs to be moved to another directory maybe /source

File: docs/system-reqs.md (hunk 1 above)
Comment (wdurairaj) [reply docs/system-reqs.md]: @prablr79 , I agree and we should'nt claim for Operating system version which are untested.

Comment (budhac) [reply docs/system-reqs.md]: Then we need to keep this updated on exact OS versions we support. Please send me that list.

File: docs/system-reqs.md [hunk 4]
```diff
@@ -3,24 +3,24 @@
...
 
-Although the plugin software is supported on these linux distributions, you should consult the Hewlett Packard Enterprise Single Point of Connectivity Knowledge (SPOCK) for HPE Storage Products for specific details about which operating systems are supported by HPE 3PAR StoreServ and StoreVirtual Storage products (https://www.hpe.com/storage/spock).
+>**NOTE:** Although the plugin software is supported on the listed Linux distributions, you should consult the Hewlett Packard Enterprise Single Point of Connectivity Knowledge (SPOCK) for HPE Storage Products for specific details about which operating systems are supported by HPE 3PAR StoreServ and StoreVirtual Storage products (https://www.hpe.com/storage/spock).
+>
+>Although other Linux distributions have not been tested, the containerized version of the plugin should work as well.
 
-NOTE: Although other linux distributions have not been tested, the containerized version of the plugin should work as well.
+**Supported software versions:**
 
-Docker EE 17.03 or later is supported.
```
Comment (prablr79) [inline docs/system-reqs.md]: Python 3.x is moved now, we support both 2.x and 3.x version, bTW by end of this year 2.x support will be stopped by Python..

File: docs/usage.md (hunk 2 above)
Comment (prablr79) [reply docs/usage.md]: @budhac sure, just want to make sure this doc is consumed in the code path.. need to do changes if required.

File: plugin-start [hunk 5]
```diff
@@ -1,19 +0,0 @@
-#!/bin/sh
```
Comment (prablr79) [inline plugin-start]: @wdurairaj do we need validate this file again ? looks like moved to different folder..

File: tox.ini [hunk 6]
```diff
@@ -1,19 +0,0 @@
-[tox]
```
Comment (prablr79) [inline tox.ini]: @imran-ansari Can you check on this file, will this affect our unit test process, looks like some files are moved to different folder

**Comment by prablr79** [comment]
@budhac I really like clean and neat Git hub page now, thanks for a good work! want to make sure that is not impacting any of the build process.  let engineers validate some of the file movements and make sure everything is intact.

**Comment by wdurairaj** [comment]
These are the list of files which needs restoration
- Dockerfile
- config/__init__.py
- paramiko_src should'nt be touched since it's part of OSRB compliance
- plugin-start
- plugin-test-start
- requirements.txt
- setup.py
- test-requirements.txt
- test/test_hpe_plugin.py
- tox.ini
- twisted/plugins/hpedockerplugin_plugin.py

These are required for the build, and deleted them will cause the CI to malfunction.

**Comment by wdurairaj** [comment]
Few Comments:
1.	https://github.com/budhac/python-hpedockerplugin talks about SuSE CaaS v3 support with 3.0 release, but actually it’s only released in 2.1. This needs correction.
2.	In the main page above, there should be a link given to paramiko src which is present in https://github.com/budhac/python-hpedockerplugin/tree/master/source/paramiko_src in README.md file for OSRB compliance
3.	Diagram in this page is missing -- https://github.com/budhac/python-hpedockerplugin/blob/master/docs/advanced/etcd_cluster_setup.md -- Possibly the image file is relocated in a different path.
a.	Original page looks like https://github.com/hpe-storage/python-hpedockerplugin/blob/master/docs/etcd_cluster_setup.md 
4.	For the supported Operating systems please give the exact version , not a specific version and later kind of comment.
5.	This link in Docker store for the 3PAR Volume Plugin points to this URL -- https://github.com/hpe-storage/python-hpedockerplugin/tree/master/quick-start , and after the PR is merged, it’s expected the quick-start folder is still accessible. But now, it will not be accessible. To correct this quick-start folder has to be created in the top of the github repository.

**Comment by prablr79** [comment]
@budhac can you close remaining items from your side..
@nilangekarss can you cross verify the build is working then lets go public.

File: docs/system-reqs.md (hunk 1 above)
Comment (prablr79) [reply docs/system-reqs.md]: @budhac its important that we don't put "latest" word until unless we qualify our work with respect to the supported OS.  Lets keep updating this information going forward.

**Pull request merged**